tests/test_mxprocessing.py

```diff
@@ -85,7 +85,7 @@ def test_processing1(testdb):
   assert rs and len(rs) >= 1
 
   # Find programs using the processing job ID and verify stored values
-  programs = mxprocessing.get_processing_job(5).programs
+  programs = testdb.get_processing_job(5).programs
```

Comment:
Not related to the commit, but: For some of the tests we have a parameter 'testconfig', but this test has 'testdb' instead, and they appear to be different things? It's a bit confusing ...

Reply:
yes, good point. Might as well do this now. I have addressed this in #63.